### PR TITLE
Improving documentation

### PR DESCRIPTION
**Pull request by vilda**

### CODE REVIEW

**Comment by vilda** [comment]
Can you have a look @StephenHynes7 @mrud ? Some proof read...

File: README.md [hunk 1]
```diff
@@ -478,3 +538,26 @@ Fields explained:
...
+combine both methods.
+
+**For small systems** such as single web server, mail server, workstation, the
+easiest way is to register the host and logs followed via the agent. The agent
+will create a Host entry in UI and all logs inside this Host. Configuration will
+be stored on Logentries systems and the agent will pull the latest configuration
+during startup.
+
+**For large systems** such as computational clusters, autoscaling setups, the
+meaning of particular host is loosing its meaning as they are becoming
```
Comment (StephenHynes7) [inline README.md]: losing

File: README.md [hunk 2]
```diff
@@ -478,3 +538,26 @@ Fields explained:
...
+ephemeral. The best option for these systems is to share the same configuration
+across servers in the cluster, using locally defined logs only with
+pull-server-side-config set to False. Logs are separated per application. All
+applications of the same type (web, mail, DB) logs into the same destination.
```
Comment (StephenHynes7) [inline README.md]: Maybe
"All applications of the same type (web, mail, DB) will send data to their own log in the Logentries UI. " ? 

Something along those lines, not sure the correct phrasing for this.

File: README.md [hunk 3]
```diff
@@ -478,3 +538,26 @@ Fields explained:
...
+Deployment best practices
+-------------------------
+
+Logentries agent provides several methods of configuration. The method you
+choose depends on the size and structure of your environment. You are free to
+combine both methods.
+
+**For small systems** such as single web server, mail server, workstation, the
+easiest way is to register the host and logs followed via the agent. The agent
+will create a Host entry in UI and all logs inside this Host. Configuration will
```
Comment (StephenHynes7) [inline README.md]: Maybe
"The agent will create a Host entry in the UI and send log data to this Host for each followed file."

**Comment by StephenHynes7** [comment]
@vilda left a few comments there.

**Comment by vilda** [comment]
Great, thanks, I'll fix those now.

**Pull request merged**